src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+import {
+    DataList,
+    NumberFormat,
+    Tag,
+    formatterUtils,
+    type IDataListItemProps,
+    type TagVariant,
+} from '../../../../../core';
+import { type ICompositeAddress } from '../../../../types';
+import { addressUtils } from '../../../../utils';
+import { MemberAvatar } from '../../../member';
+
+export interface IVoteDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The account details of the voter.
+     */
+    voter: ICompositeAddress;
+    /**
+     * The way the voter used their voting power.
+     */
+    voteIndicator: 'yes' | 'no' | 'abstain' | 'approved' | 'none';
```

Comment:
The `voting power` definition should only be applied for token-based DAOs. Also I can't see a design of Figma for `none` votes?
```suggestion
    /**
     * The vote of the user.
     */
    voteIndicator: 'yes' | 'no' | 'abstain' | 'approved';
```

Reply:
I was taking 'none' from the current app. Can remove.